<launch>
    <!-- Start the controller node -->
    <node name="rrbot_controller" pkg="ros_rrbot_lecture" type="controller.py" respawn="false" output="screen">
        <param name="hz" type="double" value="100" />
    </node>
  
</launch>